src/ioc/Context.ts

```diff
@@ -203,14 +204,37 @@ export class Context extends Lifecycle {
    * @param filelist The carried-over list of files
    * @return {Array} The list of files in this directory and subdirs
    */
-  static walkDirSync(dir, filelist = []) {
+  static walkDirLegacy(dir, filelist = []) {
```

Comment:
@pupudu ... Really good work :) I like the feature and the implementation.
I'll only ask for two things:
1.- Can you rename this method to something like walkDirNoGlob instead of walkDirLegacy? Introducing legacy in the name is a sort of signal that it will be deprecated, which is not what you're doing... you're just saying "That's the non-magic way"
2.- Add a test that covers the use of glob use-case. Should be something simple like:
```
const app = new TestInceptumApp();
app.addDirectory('/a/dir/that/has/some/files/to/be/included/and/some/not/to');
```
and then you can assert on:
```
const context = app.getContext();
context.getObjectDefinitionByName('AnObjectThatShouldExists').must.not.be.falsy();
must(context.getObjectDefinitionByName('AnObjectThatShouldNOTExists')).be.undefined();
```
or something close to that